test/torchaudio_unittest/prototype/ctc_decoder_test.py

```diff
@@ -9,21 +10,24 @@
 
 @skipIfNoCtcDecoder
 class CTCDecoderTest(TempDirMixin, TorchaudioTestCase):
-    def _get_decoder(self):
+    def _get_decoder(self, tokens=None):
         from torchaudio.prototype.ctc_decoder import kenlm_lexicon_decoder
 
         lexicon_file = get_asset_path("decoder/lexicon.txt")
-        tokens_file = get_asset_path("decoder/tokens.txt")
         kenlm_file = get_asset_path("decoder/kenlm.arpa")
 
+        if tokens == None:
+            tokens = get_asset_path("decoder/tokens.txt")
```

Comment:
is this branch necessary?

Reply:
this is just to set the default behavior if the `tokens` parameter is not passed in to this function, as in the `test_shape` test, without having a function call being used in the function header